lib/liquider/ast.rb

```diff
@@ -1,11 +1,15 @@
 module Liquider::Ast
   class Node
     class << self
-      def new_type(type_name, *attributes)
+      def new_type(type_name, *attributes, &block)
         type = Class.new(Node) do
           attributes.each do |attribute|
             attr_reader attribute
           end
+
+          def op?
```

Comment:
If I understand correctly, `op? == true` means that the node will output something?